ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -142,7 +142,7 @@ const ConfirmPageContainer = (props) => {
       fromAddress,
       global.ethereumProvider,
     );
-    setCollectionBalance(tokenBalance?.balance?.words?.[0] || 0);
+    setCollectionBalance(tokenBalance.toString(10) || '0');
```

Comment:
```suggestion
    setCollectionBalance(tokenBalance.toString() || '0');
```

The update to use the `toString` method lgtm! One thing is passing a parameter does through a warning. 
https://github.com/ethers-io/ethers.js/blob/master/packages/bignumber/src.ts/bignumber.ts#L201